baseapp/abci.go

```diff
@@ -808,6 +808,7 @@ func handleQueryApp(app *BaseApp, path []string, req abci.RequestQuery) abci.Res
 }
 
 func handleQueryStore(app *BaseApp, path []string, req abci.RequestQuery) abci.ResponseQuery {
+	const storeInfoPath = "info"
```

Comment:
1. I meant to move this globaly up top and public :p
2. Shouldn't this be `commit_info`?

Reply:
1. Will do
2. This is already under the commit substore, so the path is `/commit/info`. If I were to change it to your suggestion it would be `/commit/commit_info` which feels redundant.